web/src/main/java/com/microsoft/applicationinsights/web/extensibility/modules/WebSessionTrackingTelemetryModule.java

```diff
@@ -124,34 +110,10 @@ public void onBeginRequest(ServletRequest req, ServletResponse res) {
                 com.microsoft.applicationinsights.web.internal.cookies.Cookie.getCookie(
                         SessionCookie.class, request, SessionCookie.COOKIE_NAME);
 
-        boolean startNewSession = false;
```

Comment:
Few lines above we're checking if this a new user and set isFirst to be true. Can you please check if that also the case in the .NET after the change?